### ISSUE TITLE
Clarify which pytest items are MyPy when running with --verbose flag.

### ISSUE THREAD

**Issue by jeffbr13** [dbader/pytest-mypy#74]
When running with the `--verbose` flag, pytest-flake8 [handily annotates each output line](https://github.com/tholo/pytest-flake8/blob/c485d5e85ac6b2138405378e9763e7025f58b023/pytest_flake8.py#L90) to clarify that it ran that test (i.e. it's not a hand-written test). For example:

```sh
pytest --verbose
```
```
============================= test session starts ==============================
platform linux -- Python 3.8.1, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /usr/src/app, inifile: setup.cfg
plugins: flake8-1.0.4, mypy-0.4.2, cov-2.8.1
collecting ... collected 33 items / 2 deselected / 31 selected

Running mypy on 11 files... done with status 0
Success: no issues found in 11 source files

mymodule/__init__::py PASSED                                              [  x%]
mymodule/__init__.py::FLAKE8 PASSED                                       [  x%] <
mymodule/__main__::py PASSED                                              [  x%]
mymodule/__main__.py::FLAKE8 PASSED                                       [ x%] <
…
mymodule/utils.py::module.utils.function_name PASSED      [ x%]
…
mymodule/tests/test_function.py::test_function PASSED [ x%]
…
```

Would it be possible to similarly alter the [`MyPyItem`](https://github.com/dbader/pytest-mypy/blob/e1e4ebd49da513126afd594c96b22c0d321b9369/src/pytest_mypy.py#L84) to similarly mark type-checks as separate from hand-written tests in pytest's verbose output?

**Comment by dmtucker** [comment]
Related: #45 

This has been on my radar but lower priority. Once the xdist dust settles a bit, I'll take another look 😊

**Comment by jeffbr13** [comment]
Ahhh, apologies, I should have looked through the previous PRs a bit more closely!
